Packs/EWS/Integrations/EWSO365/EWSO365.yml

```diff
@@ -0,0 +1,1061 @@
+      required: false
+      secret: false
+    - default: false
+      description: The attachments ids to get. If none - all attachments will be retrieve
+        from the message. Support multiple attachments with comma-separated value
+        or array.
+      isArray: true
+      name: attachment-ids
+      required: false
+      secret: false
+    deprecated: false
+    description: Retrieves the actual attachments from an item (email message). To
+      get all attachments for a message, only specify the item-id argument.
+    execution: false
+    name: ews-get-attachment
+    outputs:
+    - contextPath: EWS.Items.FileAttachments.attachmentId
+      description: The attachment ID. Used for file attachments only.
+      type: string
+    - contextPath: EWS.Items.FileAttachments.attachmentName
+      description: The attachment name. Used for file attachments only.
+      type: string
+    - contextPath: EWS.Items.FileAttachments.attachmentSHA256
+      description: The SHA256 hash of the attached file.
+      type: string
+    - contextPath: EWS.Items.FileAttachments.attachmentLastModifiedTime
+      description: The attachment last modified time. Used for file attachments only.
+      type: date
+    - contextPath: EWS.Items.ItemAttachments.datetimeCreated
+      description: The created time of the attached email.
+      type: date
+    - contextPath: EWS.Items.ItemAttachments.datetimeReceived
+      description: The received time of the attached email.
+      type: date
+    - contextPath: EWS.Items.ItemAttachments.datetimeSent
+      description: The sent time of the attached email.
+      type: date
+    - contextPath: EWS.Items.ItemAttachments.receivedBy
+      description: The received by address of the attached email.
+      type: string
+    - contextPath: EWS.Items.ItemAttachments.subject
+      description: The subject of the attached email.
+      type: string
+    - contextPath: EWS.Items.ItemAttachments.textBody
+      description: The body of the attached email (as text).
+      type: string
+    - contextPath: EWS.Items.ItemAttachments.headers
+      description: The headers of the attached email.
+      type: Unknown
+    - contextPath: EWS.Items.ItemAttachments.hasAttachments
+      description: Whether the attached email has attachments.
+      type: boolean
+    - contextPath: EWS.Items.ItemAttachments.itemId
+      description: The attached email item ID.
+      type: string
+    - contextPath: EWS.Items.ItemAttachments.toRecipients
+      description: A list of recipient email addresses for the attached email.
+      type: Unknown
+    - contextPath: EWS.Items.ItemAttachments.body
+      description: The body of the attached email (as HTML).
+      type: string
+    - contextPath: EWS.Items.ItemAttachments.attachmentSHA256
+      description: The SHA256 hash of the attached email (as EML file).
+      type: string
+    - contextPath: EWS.Items.ItemAttachments.FileAttachments.attachmentSHA256
```

Comment:
are we ok wit this 5 level stuff?

Reply:
It's for BC - i.e. `EWS v2` uses the same Context Path. I believe this is something we can "live with" to keep full BC compatibility - wdyt?

Reply:
ok